apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.ts

```diff
@@ -1214,16 +1334,40 @@ export class NewLedgerEntryPanelComponent implements OnInit, OnDestroy, OnChange
         return mergedAccountTaxes;
     }
 
+    /**
+     * Prepares the voucher adjustment configuration
+     *
+     * @private
+     * @memberof NewLedgerEntryPanelComponent
+     */
+    private prepareAdjustVoucherConfiguration(): void {
+        this.adjustVoucherConfiguration = {
+            voucherDetails: {
+                voucherDate: this.blankLedger.entryDate,
+                tcsTotal: 0,
+                tdsTotal: 0,
+                balanceDue: this.currentTxn.total,
+                grandTotal: this.currentTxn.total,
+                customerName: this.currentTxn.selectedAccount ? this.currentTxn.selectedAccount.name : '',
+                customerUniquename: this.currentTxn.selectedAccount ? this.currentTxn.selectedAccount.uniqueName : '',
+                totalTaxableValue: this.currentTxn.total,
+                subTotal: this.currentTxn.total,
+            },
+            accountDetails: {
+                currencySymbol: this.currentTxn.selectedAccount ? this.currentTxn.selectedAccount.currencySymbol : this.blankLedger.baseCurrencyToDisplay.symbol
+            },
+            activeAccountUniqueName: this.activeAccount.uniqueName
+        };
+        console.log('Reached', this.currentTxn);
```

Comment:
Remove console

Reply:
Please ignore console for now as mentioned in PR comment @rohit-walkover